pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -504,4 +407,115 @@ int socket__http2_frames_parser(struct __sk_buff *skb) {
     return 0;
 }
 
+/* TLS */
+
+SEC("uprobe/http2_tls_entry")
+int uprobe__http2_tls_entry(struct pt_regs *ctx) {
+    const u32 zero = 0;
+    tls_dispatcher_arguments_t *info = bpf_map_lookup_elem(&tls_dispatcher_arguments, &zero);
+    if (info == NULL) {
+        log_debug("[http2_tls_entry] could not get tls info from map");
+        return 0;
+    }
+
+    http2_tls_state_key_t key = { 0 };
+    key.tup = info->tup;
+    key.length = info->len;
+    http2_tls_state_t *state = bpf_map_lookup_elem(&http2_tls_states, &key);
+
+    if (!state) {
```

Comment:
you have a lot of indentation here
how can it changed?

having a condition like
```c
if (state && !state->relevant) {
        bpf_map_delete_elem(&http2_tls_states, &key);
        goto exit;
}

... // rest of the code
```